docs/notes.md

```diff
@@ -0,0 +1,27 @@
+# Tracking the impact of changes
+
+Some changes in your organization, either internal or external to the environment, can impact your software delivery process. For example, changes in the team structure or membership, teams adopting new working processes, or major events affecting the system stability, might have a positive or negative impact on your teams' performance.
+
+Pulse enables you to track and correlate the impact that specific changes might have on your teams' performance using **notes** to signal actions or events that took place on a specific date.
```

Comment:
Small edit proposal just to avoid the "dangling apostrophe" (I tend to prefer that even though the text becomes longer):

```suggestion
Some changes in your organization, either internal or external to the environment, can impact your software delivery process. For example, changes in the team structure or membership, teams adopting new working processes, or major events affecting the system stability, might have a positive or negative impact on the performance of your teams.

Pulse enables you to track and correlate the impact that specific changes might have on the performance of your teams using **notes** to signal actions or events that took place on a specific date.
```